test/cmd/randstr/main.go

```diff
@@ -0,0 +1,44 @@
+package main
```

Comment:
I would really like to find a way to just use something like:
head -c XXX /dev/urandom | tr -dc .... 
but I guess it doesn't work on mac, so.... just jotting it down...